### PR TITLE
Add focus-trap and make the focus styling prettier

### LINKED ISSUE #61: A11Y: Trap focus when modal is open
**Is your feature request related to a problem? Please describe.**
When the modal is open, a user is able to tab through the webpage in the background.

**Describe the solution you'd like**
Focus should be trapped within the Uppload modal, including the "close" button.

**Describe alternatives you've considered**
N/A

**Additional context**
This helps with https://uppload.js.org/a11y

### PR DESCRIPTION
**Pull request by victorlap**
I've added a focus-trap, which fixes #61. 

One little note; When using pnpm for installing the deps, the demo would not run and complain about a missing `xtend` package. When installing with npm, this is not a problem.

### CODE REVIEW

**Comment by victorlap** [comment]
@AnandChowdhary I think there is something wrong with the snyk check here. It does not seem to relate to my commit.

**Comment by AnandChowdhary** [comment]
Thanks, @victorlap! I'll have a look at this today itself and merge it, great job! :)

Snyk is acting out, ignore it. It's using the `master` branch and looking for the wrapper `package.json` files, which don't exist anymore. I'll fix the config after this.

File: src/styles/modal.scss [hunk 1]
```diff
@@ -110,8 +110,10 @@ $modalForegroundZIndex: 12000;
           color: inherit;
           text-decoration: none;
           opacity: 0.75;
-          &:hover {
+          &:hover,
+          &:focus {
             text-decoration: underline;
+            outline: none;
```
Comment (AnandChowdhary) [inline src/styles/modal.scss]: I'm not sure about the `outline: none`, which is why I didn't add it anywhere. I find it too opinionated. Right now, Uppload links, buttons, etc., would just inherit the `:focus` states of the webpage they're running on, and maybe people want the default outlines? Would love to hear thoughts.

Comment (victorlap) [reply src/styles/modal.scss]: I added the same styling to `:focus` as is currently on `:hover`. Thus seeing the default outline is no longer necessary right?

**Comment by AnandChowdhary** [comment]
The Snyk `package.json` error is fixed now.

**Comment by victorlap** [comment]
I’ve changed the branch from this pull request. This does not seem to trigger a new round of checks though..

**Pull request merged**